tests_e2e/tests/lib/identifiers.py

```diff
@@ -18,31 +18,16 @@
 from typing import Dict, List
 
 
-class VmIdentifier(object):
```

Comment:
I merged this class into VirtualMachineClient. Makes the code less verbose.

Reply:
Looks like this file is entirely related to extension identifiers now. Should we update the name? Or do we anticipate adding non-extension related identifiers here in the future

Reply:
I thought the same thing (renaming) and laziness (or maybe another issue distracted me :)) won and I did not do it. I'll do it now.